python/doc/examples/meta_modeling/polynomial_chaos_metamodel/plot_chaos_cleaning_strategy.py

```diff
@@ -106,15 +106,15 @@
 #
 #   \mathcal{A}^{d,j} = \left\{ \boldsymbol{\alpha} \in \mathbb{N}^p
 #   \; | \; \|\boldsymbol{\alpha}\|_1 \leq d, \;
-#   \; \|\boldsymbol{\alpha}\|_0 \leq j\right\}.
+#   \; \|\boldsymbol{\alpha}\|_0 \leq j\right\}
 #
 # Therefore, the rank-`j` polynomial chaos expansion is:
 #
 # .. math::
 #
 #   h(\boldsymbol{\xi}) = \sum_{\boldsymbol{\alpha} \in
 #   \mathcal{A}^{d,j}} a_{\boldsymbol{\alpha}}
-#   \psi_{\boldsymbol{\alpha}}(\boldsymbol{\xi}) + \epsilon.
+#   \psi_{\boldsymbol{\alpha}}(\boldsymbol{\xi})
```

Comment:
This suggestion is not OK. As before, the truncation up to rank $j$ produces an expansion which is an approximation $\widetilde{h}$ of the function $h$.